app/views/chapter/show.html.haml

```diff
@@ -19,7 +19,7 @@
         - if logged_in?
           = render partial: 'subscriptions'
         - else
-          = link_to 'Sign up', new_member_path, class: 'btn btn-primary'
+          = link_to 'Join our community', new_member_path, class: 'btn btn-primary'
```

Comment:
@KimberleyCook should we standardize on either 'Join our community' or 'Join the codebar community'?